install/resources/mybb_theme.xml

```diff
@@ -822,6 +822,14 @@ blockquote cite span.highlight {
 	font-size: 13px;
 }
 
+.mycode_me {
```

Comment:
Ah, fair point. I was just trying to think of a solution so that it didn't need to be in `global.css` (hence removing the concern about theme issues). It could be that `mycode.css` is included globally? If plugins use the parser in the Admin CP that could be a problem as the stylesheet wouldn't be included there...